src/templates/Plugin.js

```diff
@@ -1,38 +1,84 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import usePageLeave from 'react-use/lib/usePageLeave';
-import Prism from 'prismjs';
+import { GatsbyImage } from 'gatsby-plugin-image';
 import { graphql } from 'gatsby';
-import { SEO, SitewideHeader, SitewideFooter, ExitIntentModal } from 'components';
+import { SEO, SitewideHeader, SitewideFooter, ExitIntentModal, Title } from 'components';
 import {
   Header,
-  Intro,
   PluginCTA,
-  CoverImage,
-  InstallationSteps,
   PlaceholderBody,
-  Notes,
   Sidebar,
+  HostTabs,
 } from 'components/backstage/plugins';
 
-// All the languages used in the plugin installation instructions need to be listed here.
-import 'prismjs/components/prism-typescript';
-import 'prismjs/components/prism-bash';
-import 'prismjs/components/prism-yaml';
```

Comment:
Now that the installation instructions are passed through the markdown processing pipeline, we no longer need to do client side syntax highlighting.